plugins/module_utils/redfish_utils.py

```diff
@@ -121,6 +121,8 @@ def patch_request(self, uri, pyld):
             if not etag:
                 etag = r['data'].get('@odata.etag')
             if etag:
+                # Clean etag and apply If-Match
+                etag = etag.strip("'\"")
```

Comment:
If you use single quotes (as above and below this snippet), you don't need the escape:
```suggestion
                etag = etag.strip('"')
```

Reply:
Here I was actually trying to capture both double and single quotes. Noting @mraineri's observation, I think it might be better to just capture the single quotes. This would still address the issue I'm running into.

Reply:
I was trying to capture both single and double quotes. Based on the comment from @mraineri, I may need to reassess the solution.